demos/random_cut_demo/src/main/scala/io/hydrosphere/rrcf/impl/RRCT.scala

```diff
@@ -146,16 +146,16 @@ class RRCT(initialData: Array[DataPoint], timeDecay: Int = 1000) {
 
         if (newCutNode.isDatapointInLeftSubtree(dataPoint)) {
           //Duplicate check
-          leafStorage.find(_.dataPoint.equals(dataPoint)) match {
+          leafStorage.get(dataPoint) match {
             case Some(duplicateLeaf) => newCutNode.left = duplicateLeaf; duplicateLeaf.childrenSize += 1

```

Comment:
in order to stay consistent, you need to remove `;` and use expression on separate lines